### PR TITLE
DHFPROD-6064: First attempt at running tests against DHS

### PR DESCRIPTION
**Pull request by rjrudin**
### Description

You'll need to add the following properties to gradle-local.properties, as the test plumbing will read properties in that file and in gradle-local.properties:

mlHost=changeme
mlUsername=changeme
mlPassword=changeme
mlSecurityUsername=same as mlUsername
mlSecurityPassword=same as mlUsername
hubDhs=true
hubSsl=true

Also deleted gradle-dhs.properties, which has no bearing on anything. 

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [ ] JIRA_ID included in all the commit messages
- [ ] PR title is in the format JIRA_ID:Title
- [ ] Rebase the branch with upstream
- [ ] Squashed all commits into a single commit
- [ ] Added Tests
  

- ##### Reviewer:

- [ ] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub/src/test/java/com/marklogic/bootstrap/TestAppInstaller.java [hunk 1]
```diff
@@ -209,6 +161,78 @@ public void run() {
         }
     }
 
+    /**
+     * When running the tests in DHS, we do not want to install DHF.
+     */
+    private void installDataHubIfNecessary() {
+        HubConfigObjectFactory factory = applicationContext.getBean(HubConfigInterceptor.class).getHubConfigObjectFactory();
+        Properties props = factory.getGradleProperties();
+        if (props.containsKey("hubDhs") && "true".equals(props.getProperty("hubDhs"))) {
```
Comment (akshaysonvane) [inline marklogic-data-hub/src/test/java/com/marklogic/bootstrap/TestAppInstaller.java:170]: Can we simplify this by using `hubConfig.getIsProvisionedEnvironment()` ?

Comment (rjrudin) [reply marklogic-data-hub/src/test/java/com/marklogic/bootstrap/TestAppInstaller.java]: Good catch, did so.

**Review by akshaysonvane — APPROVED** [review]

**Review by srinathgit — APPROVED** [review]

**Comment by srinathgit** [comment]
@rjrudin 
This PR is ready for review right ?

**Comment by rjrudin** [comment]
@srinathgit  I just rebased it locally off develop and am going to run some of the tests to see how it does.

**Pull request closed without merging**